src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int weight;
+    private int height;
+
+    public Rectangle(int weight, int height, String color) {
+        super(color);
+        this.weight = weight;
+        this.height = height;
+    }
+
+    @Override
```

Comment:
The draw() method should be implemented with a functionality to draw the rectangle